### ISSUE TITLE
Medical system not show wounds in basic and in advanced mode

### ISSUE THREAD

**Issue by Legolasindar** [acemod/ACE3#1195]
ACE3 Version: 3.0.0

**Mods:**
* `@cba_a3`
* `@ace3`

**Placed ACE3 Modules:**
One test with:
* Medical Settings
* Advanced Medical Settings

One test with:
* Medical Settings

One test with:
* Advanced Medical Settings

And one test with:
* And without moduls

**Description:**
* In basic mode, the interaction points appear with colored cross, but if you make a Diagnostic, this show all is good, no wounds (but i shoot to my patient in the arms, in the legs, in the torso).
* In advanced mode, the patient avatar no show colors of wounds, all is white, and the diagnostic table at left say all is OK, no wounds anything, but obviusly this is not true.

**Steps to reproduce:**
* Create or use not scripted mission, make one test without moduls, problem appear. I get weapon in arsenal, put BLUFOR soldier with Zeus, shoot to soldier in differents parts, and check.
* Same check with editor units (non Zeus), and with weapon from editor (non Arsenal).
* Same mission with diferents combinations of medical moduls and settings, same problem.
* Same tests with others missions.

**Where did the issue occur?**
* Occur in SP and MP mission.

**RPT log file:**
[RPT file](http://pastebin.com/DHsH6SAR)

**Comment by Glowbal** [comment]
What is the issue with advanced medical? Are the interaction actions colored but no wounds are shown on the left in the overview?

**Comment by Legolasindar** [comment]
With advanced, in the overview at left, no appear any wound, not in the listed and int in the white avatar.
![alt text](http://i.imgur.com/m5Hgbi0.png "medical bug")

**Comment by Glowbal** [comment]
Did you shoot the unit with advanced enabled? How did the unit take damage?

**Comment by Legolasindar** [comment]
I shooted with and without advanced ballistic and advanced medical.

**Comment by Glowbal** [comment]
```18:25:29   Error position: <NMD_fnc_toBool;
 
};
 
 
_return>
18:25:29   Error Variable no definida en la expresión: nmd_fnc_tobool```` 
From your rtp log.

Could you please try this with just CBA and ACE3 on a plain mission and see if it still occurs?

**Comment by Legolasindar** [comment]
The screeshot is from a plain mission with CBA and ACE3 from the editor.

**Comment by Glowbal** [comment]
Could I get the rtp log from that session?

**Comment by Legolasindar** [comment]
RPT http://pastebin.com/6N5ngQre

**Comment by Glowbal** [comment]
Did you enable advanced medical for AI?

**Comment by Legolasindar** [comment]
Ok, this is the key. With advanced medical for AI enabled, this work. Or is a bug, or error in the wiki documentation.
http://ace3mod.com/wiki/missionmaker/modules.html

```
2.2 Advanced Medical Settings

This module allows you to change the default Advanced Medical Settings, when 2.1 Medical Settings is set to "Advanced".
```
But this happen with MedicalSettings  in Normal, no in Advanced.

**Comment by Zriel88** [comment]
We are using the Advanced medical module without problems. If the screenshot is taken in editor, and the unit is an AI unit, is obvus that injures wont show on ACE3 system if it is not enabled for AI units.

**Comment by Legolasindar** [comment]
But @Zriel88 if you look the modul description, this say you need enable Advanced in Normal module, but if you try not.

**Comment by Zriel88** [comment]
This is how I have them set up,

![](http://i.gyazo.com/f38de3e7cb4ea5831d28b01df4734de0.png)

![](http://i.gyazo.com/2313c4fa0c1a76831ef071850f77d5f6.png)

The wiki says that you need point 2.1 of Medical settings in "advanced" (which corresponds with Medical Level), in order to use the advanced medical system, which is correct.
Then you have the "Advanced Medical Settings" module, in which you tweak the settings for advanced medical system.

The problem here is that injuries were not showing for you in editor. which is correct if you don't enable them for AI too. Default is players only.

**Comment by Legolasindar** [comment]
This is not the point.
The wiki says that you need **Medical settings** in "Advanced", BUT if i set Normal, same of this picture.

![Medical Settings Level in Normal](http://i.imgur.com/UGeHOms.png "Medical Settings Level in Normal")

And you set the **Advanced Medical Settings** in **Player only**, some of this picture.

![Advanced medical in Player only](http://i.imgur.com/FDOQJDt.png "Advanced medical in Player only")

When you injure a IA, and select diagnosis in the Head, this says all is OK, but this is not true (i see IA bleeding).

And if you set **Advanced Medical Settings** in **Player and IA**, some of this picture.

![Advanced medical in Player and IA](http://i.imgur.com/MD1CJW4.png "Advanced medical in Player and IA")

Now when you make a diagnostic in the Head, you can see all injures.

The option **Enabled for** in the Advanced Medical Settings is working whether in Medical Settings is activated or not the Advanced mode.

**Comment by Zriel88** [comment]
Well, it is just that the settings for enabling the medical system for AI are on the advanced medical settings module.
Maybe it should be moved to the Medical Settings module, but I don't think it is really a bug, more like an improvement.

**Comment by Legolasindar** [comment]
This is my point, or is a Bug, and need to be fixed, or is a wrong wiki information, and need be fixed in the Wiki ;)

**Comment by Glowbal** [comment]
Thanks @Legolasindar It is now fixed in #1242

**Comment by Legolasindar** [comment]
You're welcome, and sorry for my bad english explanation ;)
